src/main/java/core/basesyntax/service/serviceimpl/DataConverterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public static final String SEPARATOR = ",";
+    public static final int REPORT_START_POSITION_INDEX = 1;
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_NAME_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
```

Comment:
Consider moving these constants into a configuration class or enum as they are related to the parsing of CSV data and may be used in other parts of the application as well.